rollup.config.js

```diff
@@ -0,0 +1,77 @@
+
+import nodeResolve from 'rollup-plugin-node-resolve';
+import commonjs    from 'rollup-plugin-commonjs';
+
+
+
+
+
```

Comment:
That is definitely the violation of the current lint rules. This file should conform them

Reply:
will fix

Reply:
> That is definitely the violation of the current lint rules. This file should conform them

Fixed, but, there are other violations in here, such as that strings are still single quote

I need to fix the disparity between the codebase being `commonjs` and the rollup being `es6` before the linter will lint correctly

As a result I've fixed this with the intent to finish lint fixing after I've fixed the linting tool itself